src/main/java/com/iota/iri/Snapshot.java

```diff
@@ -34,7 +34,7 @@ public static Snapshot init(SnapshotConfig config) throws IOException {
             String snapshotFile = config.getSnapshotFile();
             if (!config.isTestnet() && !SignedFiles.isFileSignatureValid(snapshotFile, config.getSnapshotSignatureFile(),
                     SNAPSHOT_PUBKEY, SNAPSHOT_PUBKEY_DEPTH, SNAPSHOT_INDEX)) {
-                throw new RuntimeException("Snapshot signature failed.");
+                throw new IOException("Snapshot signature failed.");
```

Comment:
Throw specific excetion instead of a generic one

Reply:
I disagree. Is an invalid signature an IOException?